build/dependencies.props

```diff
@@ -56,7 +56,7 @@
     <MicrosoftNETCoreApp21PackageVersion>2.1.0-preview2-26130-04</MicrosoftNETCoreApp21PackageVersion>
     <MicrosoftNETTestSdkPackageVersion>15.3.0</MicrosoftNETTestSdkPackageVersion>
     <MoqPackageVersion>4.7.49</MoqPackageVersion>
-    <MsgPackCliPackageVersion>0.9.0-beta2</MsgPackCliPackageVersion>
+    <MsgPackCliPackageVersion>1.0.0-rc</MsgPackCliPackageVersion>
```

Comment:
We have to do something in universe for this I think

Reply:
Ugh. Yeah probably